firestore-genai-chatbot/functions/src/generative-client/google_ai.ts

```diff
@@ -14,7 +14,8 @@ interface GeminiChatOptions {
   projectId: string;
   location: string;
   context?: string;
-  safetySettings: SafetySetting[];
+  //TODO where to get these from Genkit?
+  // safetySettings: SafetySetting[];
```

Comment:
https://github.com/firebase/genkit/blob/next/js/plugins/vertexai/src/gemini.ts#L71

I think you can provide a `config` object